### PR TITLE
2831 update preferred language flow

### PR DESCRIPTION
**Pull request by shawn320**
The purpose of this pr is to refactor the preferred language flow. The edit page should now redirect to the new confirmation page, which will update preferred language through service then redirect to personal info page.

### CODE REVIEW

File: frontend/public/locales/fr/personal-information.json [hunk 1]
```diff
@@ -99,7 +109,7 @@
       "change": "(FR) Change"
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/personal-information.json:109]: **Suggestion:** With `_gcweb-app.personal-information.preferred-language._index.tsx` deleted, I feel these translations can be removed as well.

File: frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    const userId = await userService.getUserId();
+    const session = await sessionService.getSession(request.headers.get('Cookie'));
+
+    const preferredLanguage = await session.get('preferredLanguage');
+    await userService.updateUserInfo(userId, { preferredLanguage });
+
+    return redirect('/personal-information');
+  } catch (error) {
+    console.error(error);
+  }
```
Comment (nicholas-ly) [inline frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx:44]: **Suggestion:** We can remove the `try`/`catch` as there's no need to output the error in the console. In the case of any errors in this function, the application will log them and render the `ErrorBoundary` as a result.

File: frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  return (
+    <>
+      <h1 id="wb-cont" property="name">
+        {t('personal-information:preferred-language.confirm.page-title')}
+      </h1>
+      <p>{t('personal-information:preferred-language.confirm.confirm-message')}</p>
+      <Form method="post">
+        <div className="form-group">
+          <dl>
+            <dt>{t('personal-information:preferred-language.index.language')}</dt>
```
Comment (nicholas-ly) [inline frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx:59]: **Nitpick:** The WET design guide suggests wrapping labels and form controls in a `form-group`. See https://wet-boew.github.io/wet-boew-styleguide/design/forms-en.html#basic

I feel the `<div className="form-group">` can be removed as it wraps a `<dl>` and not a form control element.

**Review by Fbasham — APPROVED** [review]
Overall great job.  It builds and runs fine and conforms to the other detail pages ux/flow.  I was curious about the semantics of the form, but it passes an HTML validator check via W3C and doesn't appear to have any a11y issues so we should be good to go.

**Review by nicholas-ly — COMMENTED** [review]
Flow looks good to me. 

I made a few comments but none I feel should block this PR. Feel free to address them in a future PR by resolving my comments yourself.

File: frontend/public/locales/fr/personal-information.json (hunk 1 above)
Comment (nicholas-ly) [reply frontend/public/locales/fr/personal-information.json:109]: By "these translations", i mean all translations within `index`.

**Pull request merged**